ocs_ci/ocs/platform_nodes.py

```
@@ -4249,3 +4262,504 @@ def disable_nodes_network_temporarily(self, nodes, duration=20):
+        from ocs_ci.helpers.helpers import refresh_oc_login_connection
+
+        logger.info(
+            "Teardown: Ensuring all agent hostedcluster nodes are powered on "
+            "and cluster is accessible"
+        )
+
+        try:
+            try:
+                refresh_oc_login_connection()
+                logger.info("Successfully re-authenticated to hostedcluster")
+            except Exception as login_error:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)